setup.cfg

```diff
@@ -10,9 +10,13 @@ junit_logging = all
 junit_family = xunit2
 
 [flake8]
+max-line-length = 88
```

Comment:
I think that belongs originally to https://github.com/RIOT-OS/Release-Specs/pull/233 ;-)?

Reply:
Yes and no, I will run into this anyway when reformatting now.

Reply:
Without this after the automatic formating I get these extra flgs:

```
flake8 installed: flake8==3.9.2,mccabe==0.6.1,pycodestyle==2.7.0,pyflakes==2.3.1
flake8 run-test-pre: PYTHONHASHSEED='3445153151'
flake8 run-test: commands[0] | flake8
./07-multi-hop/test_spec07.py:208:80: E501 line too long (82 > 79 characters)
./09-coap/test_spec09.py:99:80: E501 line too long (82 > 79 characters)
./testutils/github.py:127:80: E501 line too long (83 > 79 characters)
./testutils/github.py:371:80: E501 line too long (87 > 79 characters)
./testutils/iotlab.py:53:80: E501 line too long (85 > 79 characters)
./testutils/iotlab.py:92:80: E501 line too long (84 > 79 characters)
./testutils/iotlab.py:135:80: E501 line too long (88 > 79 characters)
./testutils/pytest.py:59:80: E501 line too long (82 > 79 characters)
./testutils/pytest.py:84:80: E501 line too long (84 > 79 characters)
./testutils/pytest.py:89:80: E501 line too long (83 > 79 characters)
./testutils/shell.py:171:80: E501 line too long (84 > 79 characters)
./08-interop/test_spec08.py:44:80: E501 line too long (80 > 79 characters)
./08-interop/test_spec08.py:82:80: E501 line too long (87 > 79 characters)
```

Reply:
So I applied right away since we well using black afterward anyway for formatting

Reply:
IMHO the line lengths should be fixed properly... This change is unrelated. There are disable comments [suddenly appearing](https://github.com/RIOT-OS/Release-Specs/pull/234/files#diff-a31c7ed5d35f5ed8233994868c54d625b18e6bacb6794344c4531e62bd9dde59R35) where the script made no change and still they are there.